tests/core/creators/detached.js

```diff
@@ -0,0 +1,150 @@
+/* bender-tags: editor, 4.17.0, feature, 4461*/
```

Comment:
We use version tagging only for manual tests.

Reply:
There are some exceptions (like `widgetselection`) ;p  but I remove it :)

Reply:
Exceptions confirm the rule :P In manuals we use version tags to filter tests during testing phase, but we run all the units anyway, so it is a waste of a few seconds 😜  And if someone wants to add another test to this file in different release, this tag won't be valid anymore (same for the ticket reference btw), while we never add more cases to existing manual tests so their tags don't change.